packages/components/src/select-dropdown/README.md

```diff
@@ -0,0 +1,236 @@
+# Select Dropdown
```

Comment:
The only contention I have with this issue is that we loose the blame history of the file once we delete that.

Reply:
Ah this is a super interesting point that I also don't have a great answer to. For me, being able to break down the migration into easily digestible PRs is worth the trade off, especially for such a wide reaching component.

At the very least though, I'll go back and make a note in the pull request descriptions for this PR, https://github.com/Automattic/wp-calypso/pull/79256, and https://github.com/Automattic/wp-calypso/pull/79257 if anyone else follows this same process